tests/assembly/x86_64-windows-float-abi.rs

```diff
@@ -37,7 +37,7 @@ pub extern "C" fn second_f64(_: f64, x: f64) -> f64 {
 }
 
 // CHECK-LABEL: second_f128
-// CHECK: movaps %xmm1, %xmm0
+// CHECK: movaps {{(%xmm1|\(%rdx\))}}, %xmm0
```

Comment:
Could you add something like `// FIXME(llvm21) remove xmm1 once Rust's LLVM has the ABI change`  so we clean this up later?

Cc @beetrees since iirc you authored this test

Reply:
Actually if we update our codegen to match `i128`, the options shouldn't be needed since it will always be `(%rdx)`.

Reply:
Done, and revised the PR title/summary to match the new commit structure.